ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -147,6 +153,16 @@ module.exports = Marionette.LayoutView.extend({
     this.determineInput()
   },
   transformValue: function(value, comparator) {
+    const numberComparators = ['>', '<', '=', '>=', '<=']
```

Comment:
Why do we make a new array here instead of using the one defined up top?

Reply:
This one doesn't have the EMPTY string in the object, but I will make changes to this line.

Reply:
Ah we're in a view here and we don't want to display that. I lost track of that.